### ISSUE TITLE
Snapshot process never finishes

### ISSUE THREAD

**Issue by dfosco** [hyperspace/hyperspace#15]
First time it failed because phoenix.app wasn't authorized for accessibility features, but now it won't go regardless :/

I'm on MacOS version `10.12.3`
npm version `4.1.2`
node version `7.7.4`

![image](https://cloud.githubusercontent.com/assets/3229933/26063979/f9b9c536-3965-11e7-836b-3999c77d7d05.png)
